apis/nucleus/src/components/listbox/ListBoxPopover.jsx

```diff
@@ -64,8 +65,9 @@ export default function ListBoxPopover({
               qSortByLoadOrder: 1,
             },
           ],
-          qFieldDefs: [fieldName],
+          ...(!isMasterDim ? { qFieldDefs: [fieldName] } : {}),
```

Comment:
Maybe it just me, but in find the suggestion below easier to read. I would be supprised if it didn't work when pass in `undefined` for an optional value. Either way comment a do as you want

```suggestion
          qFieldDefs: isMasterDim ? undefined : [fieldName],
```

Reply:
Yes, I was thinking about `undefined` but I think this is **more robust** because in falsy situations, it does not pass the property at all.

Reply:
I guess if they used the `in` operator it would not work